client/my-sites/patterns/components/pattern-gallery/style.scss

```
@@ -1,8 +1,29 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
 .pattern-gallery {
 	display: grid;
 	gap: 48px;
 }
 .pattern-gallery--grid {
 	gap: 16px 24px;
-	grid-template-columns: repeat(3, 1fr);
+
+	@include break-medium {
+		grid-template-columns: repeat(2, 1fr);
+	}
+
+	@include break-huge {
+		grid-template-columns: repeat(3, 1fr);
+	}
+}
+.pattern-gallery--grid.pattern-gallery--pages {
```

NIT let's add empty lines in this module
```suggestion

.pattern-gallery--grid.pattern-gallery--pages {
```